### PR TITLE
Power BI - Add management code sample

### PR DESCRIPTION
**Pull request by simonaco**

### CODE REVIEW

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/powerbi-embedded.md](https://github.com/simonaco/azure-docs-sdk-node/blob/patch-7/docs-ref-services/powerbi-embedded.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C15%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707151707143871-91%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/powerbi-embedded.md [hunk 1]
```diff
@@ -13,26 +13,59 @@ ms.devlang: nodejs
...
 
-### Management
 ```bash
 npm install azure-arm-powerbiembedded
 ```
 
+### Example
 
-##Example
+Authenticate, create client and create workspace collection in an existing resource group.
```
Comment (johnpapa) [inline docs-ref-services/powerbi-embedded.md]: missing "the".

> create the client

> create the workspace collection

File: docs-ref-services/powerbi-embedded.md [hunk 2]
```diff
@@ -13,26 +13,59 @@ ms.devlang: nodejs
...
+```javascript
+const msRestAzure = require('ms-rest-azure');
+const PowerBIEmbeddedManagementClient = require('azure-arm-powerbiembedded');
 
-Explore more [sample Node.js code](https://azure.microsoft.com/resources/samples/?platform=nodejs) you can use in your apps.
\ No newline at end of file
+const creationOptions = {
+  location: 'northcentralus',
+  tags: {
+    key1: 'value1',
```
Comment (johnpapa) [inline docs-ref-services/powerbi-embedded.md]: what are the keys for? would a better variable name help clarify it? if not, maybe a small comment.

File: docs-ref-services/powerbi-embedded.md [hunk 3]
```diff
@@ -13,26 +13,59 @@ ms.devlang: nodejs
...
 
 With the Power BI Embedded Azure service, you can integrate Power BI reports right into your node application to create or edit charts and reports.
 
 Learn more about [Power BI Embedded](https://powerbi.microsoft.com/en-us/documentation/powerbi-developer-embedding/).
 
-## Install the modules with npm
+## Management Package
 
-Use npm to install the Azure PowerBI Embedded module for Node.js
+### Install npm module
```
Comment (johnpapa) [inline docs-ref-services/powerbi-embedded.md]: Missing the 

> Install the Azure Power BI npm module

File: docs-ref-services/powerbi-embedded.md [hunk 4]
```diff
@@ -13,26 +13,59 @@ ms.devlang: nodejs
...
+    teir: 'Standard'
+  }
+};
+
+msRestAzure
+  .interactiveLogin()
+  .then(credentials => {
+    const client = new PowerBIEmbeddedManagementClient(
+      credentials,
+      'your-subscription-id'
```
Comment (johnpapa) [inline docs-ref-services/powerbi-embedded.md]: Let's move all variable up top like this ....


```javascript
const msRestAzure = require('ms-rest-azure');
const PowerBIEmbeddedManagementClient = require('azure-arm-powerbiembedded');

const subscriptionId =  'your-subscription-id';
const resourceGrop = 'your-resource-group-name';
const workspace = 'workspace-collection-name';

// ....
```

**Comment by johnpapa** [comment]
added intros for heading sections

needs review before #sign-off

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/powerbi-embedded.md](https://github.com/simonaco/azure-docs-sdk-node/blob/patch-7/docs-ref-services/powerbi-embedded.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180059591072-91%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/powerbi-embedded.md](https://github.com/simonaco/azure-docs-sdk-node/blob/patch-7/docs-ref-services/powerbi-embedded.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180410166445-91%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/powerbi-embedded.md](https://github.com/simonaco/azure-docs-sdk-node/blob/patch-7/docs-ref-services/powerbi-embedded.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180422478871-91%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by johnpapa** [comment]
LGTM

#sign-off

**Pull request merged**